### ISSUE TITLE
mod completely breaks in multiplayer

### ISSUE THREAD

**Issue by shamshitty** [ReplayMod/ReplayMod#735]
I spent a few minutes recording on a friend's server only to find out that the recording gets stuck in an infinite loading loop. the world doesn't even load, it just says "joining world" for a few seconds before saying "loading terrain" for like one frame.

in singleplayer the mod works absolutely fine with no problems, but when I view a multiplayer recording _everything_ breaks and I get softlocked.

_using hypixel as an example of what I see_
![ezgif com-gif-maker](https://user-images.githubusercontent.com/68004605/169689699-dac26c44-4071-451d-a6b5-bc542a4d3db7.gif)

**Comment by Johni0702** [comment]
Please reproduce the issue and then attach the `latest.log` file.

**Comment by shamshitty** [comment]
here's the log file
[latest.log](https://github.com/ReplayMod/ReplayMod/files/8754593/latest.log)

**Comment by shamshitty** [comment]
So by disabling every mod installed _(apart from iris, sodium, and all that since the version of RM I use requires it)_ and trying to play the replay, I got the same result. So I recorded a new replay with all mods off (with iris, etc still on) and it worked fine. Enabled mods one by one with no issues. Even with every single mod enabled and playing the new replay, there were no issues.

So something goes wrong _during_ recording rather than playback.
